captcha/widgets.py

```diff
@@ -58,5 +58,6 @@ def get_context(self, name, value, attrs):
             'public_key': self.public_key,
             'lang': lang,
             'options': mark_safe(json.dumps(self.attrs, indent=2)),
+            'attrs': self.attrs,
```

Comment:
I think it may be better to handle this in the `except AttributeError` block above. Could you please modify it to be as follows:
```
try:
    context = super(ReCaptcha, self).get_context(name, value, attrs)
except AttributeError:
    context = {
        "widget": {
            "attrs": self.build_attrs(attrs)
        }
    }
```